README.md

```diff
@@ -292,6 +292,27 @@ When executed with the `--format=json` option, you can output list in JSON forma
 
 ```
 
+#### JSON LicenseFinder
+
+When executed with the `--format=json-license-finder` option, you can output list in JSON format that is identical to [LicenseFinder](https://github.com/pivotal/LicenseFinder). The `jlf` keyword is prepared as alias of `jlf`.
+This makes pip-licenses a drop-in replacement for LicenseFinder.
+
+```json
+[
+  {
+    "licenses": ["BSD"],
+    "name": "Django",
+    "version": "2.0.2"
+  },
+  {
+    "licenses": ["MIT"],
+    "name": "pytz",
+    "version": "2017.3"
+  }
+]
+
+```
+
```

Comment:
Great documentation 👍 

This Markdown file has a table of contents at the top. Can you add a link to the table of contents for this option?

Reply:
Added :)